src/modules/Token.js

```diff
@@ -0,0 +1,13 @@
+import { DataTypes } from 'sequelize';
+import { client } from '../utils/db.js';
+import { User } from './User.js';
+
+export const Token = client.define('token', {
+  refreshToken: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+});
+
+Token.belongsTo(User);
+User.hasOne(Token);
```

Comment:
The relationship `User.hasOne(Token)` might not be appropriate if a user is expected to have multiple tokens. Consider using `User.hasMany(Token)` to reflect a one-to-many relationship.